pos_journal_pin/README.rst

```diff
@@ -6,9 +6,7 @@
  POS Payments by Manager's PIN
 ===============================
 
-Ask for manager permission before use the journal
-
-The module requires managers PIN to be entered to use the journal
+The module requires managers permission, which can be attained by entering PIN prior the usage of the journal. 
```

Comment:
I think it would be better to specify that it is the **POS** managers permission. Then it'll have a meaning of the special group of users. Here and in other files too

Reply:
fixed in all related files